packages/portal/spec-renderer/src/components/operations-list/OperationsListItem.vue

```diff
@@ -1,12 +1,12 @@
 <template>
   <button
-    class="item"
+    class="spec-operations-list-item"
     :class="{
       'item--selected': isSelected,
       [`item--method-${item.method}`]: true,
       'disabled': disableSelection
     }"
-    :data-testid="`spec-operations-list-item-${item.path}`"
+    :data-testid="`spec-operations-list-item-${item.method.toLowerCase()}${item.path.replaceAll('/', '-')}-${item.tag || '-'}`"
```

Comment:
suggestion: this attribute has a lot of logic; worth moving to a computed variable?